Assets/Scripts/NavigationArrow.cs

```diff
@@ -69,6 +71,12 @@ private NavObject GetNavObjectToFollow(NavObject navObject)
 
     private void SetupNavObjects()
     {
+        foreach (Transform child in navObjectParent.transform)
+        {
+            navObjects.Add(child.gameObject.GetComponent<NavObject>());
+        }
+        navObjects.Reverse();
+
         for (int iterator = 0; iterator < navObjects.Count; iterator++)
         {
             navObjects[iterator].id = iterator;
```

Comment:
Can we combine these two?

Reply:
No

Reply:
why not? It seems like both for loops get access to each individual NavObject right? So it could set up the ids and listeners to them too 🤔